src/components/Container/index.css

```
@@ -0,0 +1,6 @@
+.round-square {
```

Rename it to camelCase so that you can access it inside the component with 
`styles.roundSquare` instead of `styles['round-square']`.

```suggestion
.roundSquare {
```